pac4j-saml/src/main/java/org/pac4j/saml/sso/impl/SAML2DefaultResponseValidator.java

```diff
@@ -674,7 +674,12 @@ protected final void validateSignature(final Signature signature, final String i
     private boolean isDateValid(final DateTime issueInstant, final int interval) {
         final DateTime before =  DateTime.now().plusSeconds(acceptedSkew);
         final DateTime after =  DateTime.now().minusSeconds(acceptedSkew + interval);
-        return issueInstant.isBefore(before) && issueInstant.isAfter(after);
+        boolean isDateValid = issueInstant.isBefore(before) && issueInstant.isAfter(after);
+		if (logger.isTraceEnabled() && !isDateValid) {

```

Comment:
I always do that for performance reasons: http://learnertobeginner.blogspot.com/2010/06/how-to-use-isdebugenabled-effectly.html. Shall I remove it as requested?